### PR TITLE
Open folder button for backup

### PR DESCRIPTION
**Pull request by Grayyyyy**
Issue https://github.com/lbryio/lbry-app/issues/1601 resolved
also yes I know I accidentally typed "backp" in the title of the commit lol
Pretty sure this works cross-platform, but testing on Mac and Linux would be recommended

### CODE REVIEW

**Comment by tzarebczan** [comment]
Thanks for this PR @Grayyyyy ! We are prepping a redesign release, so not sure we'll get to this before then - thanks for your patience!

**Comment by seanyesmunt** [comment]
This doesn't seem to work (well) for mac. Since `.lbryum/` is a hidden folder, it just opens up the file explorer. I have to manually show hidden files before I can see that `.lbryum/` is selected. Not sure if there is a way around this.

File: src/renderer/page/backup/view.jsx [hunk 1]
```diff
@@ -37,9 +38,21 @@ class BackupPage extends React.PureComponent<Props> {
                 </p>
                 <p>
                   {__(
-                    'However, it is fairly easy to back up manually. To backup your wallet, make a copy of the folder listed below:'
+                    'However, it is fairly easy to back up manually. To backup your wallet, click the button below to open your wallet directory, and copy the files to a safe location.'
                   )}
                 </p>
+				<div className="walletbackup__actions">
```
Comment (seanyesmunt) [inline src/renderer/page/backup/view.jsx]: This div isn't needed.

File: src/renderer/page/backup/view.jsx [hunk 2]
```diff
@@ -37,9 +38,21 @@ class BackupPage extends React.PureComponent<Props> {
...
+                <Button
+                  button="primary"
+                  label={__('Open Wallet Directory')}
+                  onClick={() => shell.showItemInFolder(lbryumWalletDir)}
+                />
+                </div>
+		          <p>
```
Comment (seanyesmunt) [inline src/renderer/page/backup/view.jsx]: Not sure why the formatting is off for this `<p>` tag. [Prettier](https://github.com/prettier/prettier) should run automatically when you commit.

**Comment by seanyesmunt** [comment]
I'm gonna close this since the same code exists in https://github.com/lbryio/lbry-app/pull/1638

**Pull request closed without merging**